### ISSUE TITLE
investigate docker

### ISSUE THREAD

**Issue by BurntSushi** [BurntSushi/nfldb#48]
I've been playing around with [docker](https://www.docker.com) at work, and if it can be made easy to run images on Windows, it might be a great idea to use it to distribute `nfldb`. The image would include PostgreSQL and the database itself. Possibly even the Python module.

This would be great for users that just want a database up and running that they can query.

**Comment by mcd0well** [comment]
Hi, did you make any progress with this? I am relatively new to docker, but I am thinking about trying to do something similar as an exercise in itself.

**Comment by BurntSushi** [comment]
No. The right way to do this is to probably build one image that has `nfldb` installed and distribute a `docker-compose.yml` file that runs two containers: one that runs `nfldb-update` (using the `nfldb` image) and one that runs PostgreSQL (using the standard `postgres` image).

**Comment by alicona1** [comment]
First time to put something up on github, but this season I've been using [this docker setup](https://github.com/alicona1/nfldb-docker) for NFLDB.
